authors/apps/profiles/tests/test_list_profiles.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.views import status
+from ...articles.tests.base_test import TestBaseCase
+
+
+class TestListUserProfiles(TestBaseCase):
+    def test_authenticated_user_list_profiles(self):
+        self.signup_user()
+        self.signup_user2()
```

Comment:
Hey @JoshuaKodhe, I just want to understand why you have to sign up two users at this instance

Reply:
> Hey @JoshuaKodhe, I just want to understand why you have to sign up two users at this instance

The first user will be the one to log in(authenticated user), the second user is to represent other users on the system. The current logged in user will be excluded from the list(my assumption is that they already know their profiles).  so the second user will be returned in this case.